src/material-experimental/mdc-menu/menu.ts

```diff
@@ -56,11 +57,22 @@ export class MatMenu extends _MatMenuBase {
   protected override _elevationPrefix = 'mat-mdc-elevation-z';
   protected override _baseElevation = 8;
 
+  /*
+   * @deprecated `changeDetectorRef` parameter will become a required parameter.
+   * @breaking-change 15.0.0
+   */
+  constructor(
+    elementRef: ElementRef<HTMLElement>,
+    ngZone: NgZone,
+    defaultOptions: MatMenuDefaultOptions,
+  );
+
```

Comment:
Just checking, would we need this deprecation for the MDC-based menu as well?

Reply:
Both the MDC and non-MDC menus have it, in case the MDC one is moved into stable before we've cleaned up the deprecation.

Reply:
Fair enough. I was thinking that we are good with such breaking changes when we make the MDC one the "standard" one